indico/modules/events/registration/controllers/management/tickets.py

```diff
@@ -73,6 +74,11 @@ def _process(self):
                 'scope': 'registrants',
             }
         }
+        if ticker_qr_code_handlers := get_custom_ticket_qr_code_handlers():
+            qr_data['regex'] = []
```

Comment:
s/ticker/ticket/ and I would always add this list. Maybe call it `custom_code_handlers` instead?